blog/2024-01-30-FAQ.md

```diff
@@ -95,6 +95,23 @@ You can close your position, but your channel will remain open until you decide
 
 Depending on the size of this channel, you'll be able to open larger or smaller positions in the trading section, or hold more or less synthetic currency in your wallet (to be released)
 
+### Why my position has an expiry?
+
+This is how futures contracts work, and so are discreet log contracts.
```

Comment:
True, but we usually talk about perpetual swaps, which do not have fixed expiries.

I would say in an ideal world there would be no expiry, but it is the only way that we can simulate a perpetual swap using DLCs. DLCs are open until the oracle attests to the outcome of an event (attestation time). That's why we have a rollover protocol which replaces the old DLC with a new one with an attestation time further in the future.

Reply:
Nice, thanks for including the feedback!

I would then delete line 100, since the paragraph below is already answering the question (and it's not completely correct).